src/main/java/io/opencensus/integration/jdbc/Observability.java

```diff
@@ -44,6 +44,7 @@
 import java.util.Map;
 
 public final class Observability {
+
```

Comment:
Spurious line or is this Java style?

Reply:
That's how java likes one empty line there.